clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -151,6 +152,10 @@ static bool useFramePointerForTargetByDefault(const llvm::opt::ArgList &Args,
     }
   }
 
+  if (toolchains::isARMBareMetal(Triple)) {
+    return false;
+  }
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
done!